catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -402,4 +422,14 @@ protected JdbcColumn.Builder getBasicJdbcColumnInfo(ResultSet column) throws SQL
         .withNullable(nullable)
         .withDefaultValue(defaultValue);
   }
+
+  // Different JDBC catalog may have different escape character, please overwrite this method if
+  // necessary
+  protected String escapeSQL(String s) {
```

Comment:
escapeSQLIdentifier?

Reply:
I afraid this is not called an `identifier`, like `where a = 'xxxxx'`, `xxxxx` is not an identifier strictly.

Reply:
Do you have any other ideas about it? 
@diqiu50 @mchades

Reply:
`replaceWildcards` ?

Reply:
Whether the table name will contain dots `.`？

Reply:
"Different JDBC catalogs may have different escape characters, please override this method if required"

Reply:
This method has been removed.